src/sentry/static/sentry/app/views/organizationIntegrations/integrationRequest/RequestIntegrationButton.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {openModal} from 'app/actionCreators/modal';
+import Button from 'app/components/button';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+
+import {RequestIntegrationProps} from '.';
+import RequestIntegrationModal from './RequestIntegrationModal';
+
+type Props = RequestIntegrationProps;
+type State = {
+  isOpen: boolean;
+  isSent: boolean;
+};
+
+export default class RequestIntegrationButton extends React.Component<Props, State> {
+  constructor(props: Props, context) {
+    super(props, context);
+
+    this.state = {
```

Comment:
@mgaeta You can always simplify this to something like:
```
export default class RequestIntegrationButton extends React.Component<Props, State> {
    state = {
      isOpen: false,
      isSent: false,
   };
```

No need to re-implement the constructor to set initial state.